action/execution.go

```diff
@@ -78,6 +82,44 @@ func (ex *Execution) Data() []byte { return ex.data }
 // Payload is same as Data()
 func (ex *Execution) Payload() []byte { return ex.data }
 
+// AccessList returns the access list
+func (ex *Execution) AccessList() types.AccessList { return ex.accessList }
+
+func toAccessListProto(list types.AccessList) []*iotextypes.AccessTuple {
+	if len(list) == 0 {
+		return nil
```

Comment:
Is it ok to return `nil` instead of empty slice?

Reply:
yeah, there is test case with `list = nil`